File: string/string7.cpp
// C++ program to count all the vowels in a given string. 

#include <iostream>
#include <vector>
#include <string>
#include <algorithm>

int main(){
    
    std::string input = "eagerer"; 
    
    std::string vowel = "aiueoAIUEO";    

    int count = 0;
    for(int i = 0; i<input.size(); i++){
        auto it = std::find(vowel.begin(), vowel.end(), input[i]);
            if(it != vowel.end()){
                std::cout << input[i] << std::endl;
                count += 1;
            }
        
    } 

    std::cout << count << std::endl;

}

File: string/string7.go
package main

import (
	"fmt"
)

var input string = "eagerer"; 
var vowels string = "aiueoAIUEO";
var count int = 0;


func find(char rune, vowels string) bool{
	for _, vowel := range vowels{
		if char == vowel{
			return true;
		}
	}
	return false;
}

func main(){

	for _, char := range input {
		
		if find(char, vowels){
			count++;
		}
	}  

	fmt.Printf("%d \n", count);

}